app/assets/stylesheets/providers/_check-your-answers.scss

```diff
@@ -0,0 +1,88 @@
+@import "govuk-frontend/settings/all";
```

Comment:
Just a couple of questions:

* Is there any particular reason for having the filename with a prefixed underscore? Also, I would suggest we keep the filename to the more conventional `check_your_answers.scss`. 
* Should all the imports set here already been imported by the global [application.scss](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/master/app/assets/stylesheets/application.scss#L24) one?

Reply:
I just copied and pasted the file from the GDS Prototype Kit, where it had an underscore. I've removed it for consistency.

The styling of the page fails if those imports aren't included. I discussed it with someone at GDS on x-gov slack and they confirmed that they were explicitly needed.